doc/plugins.md

```diff
@@ -300,6 +300,8 @@ The list of plugins:
     — configure remark w/ YAML
 *   🟢 [`remark-code-title`](https://github.com/kevinzunigacuellar/remark-code-title)
     — add titles to code blocks
+*   🟢 [`remark-flexible-code-titles`](https://github.com/ipikuka/remark-flexible-code-titles)
+    — add titles or/and containers for code blocks with customizable attributes
```

Comment:
Please sort this item. It should be above `remark-frontmatter`

Reply:
Okey.